zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/cache/ProcessCacheItems.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.cache;
+
+import java.util.Map;
+
+public record ProcessCacheItems(Map<Long, ProcessCacheItem> processCacheItems) {
+  public String getFlowNodeName(final Long processDefinitionKey, final String key) {
+    final var processCacheItem = processCacheItems.get(processDefinitionKey);
+    return processCacheItem != null ? processCacheItem.getFlowNodeName(key) : key;
```

Comment:
❓ Can we use the `getOrDefault` method of the `Map` interface here? If so, do we need this extra class?

Reply:
There are 2 maps, first returns the `ProcessCacheItem` the second the node name.
So, we could probably use the `getOrDefault` but it needs to return a `ProcessCacheItem`.
Maybe something like this:

```
processCacheItems.getOrDefault(ProcessCacheItem.empty()).getFlowNodeName(key);
```

> If so, do we need this extra class?

We don't, I just thought it was nicer this way.

Reply:
Dropped it and replaced with `getOrDefault`s.